include/dev.h

```diff
@@ -37,12 +37,13 @@
 #define MCH_PCI_DEVICE		0x18
 #define MCH_PCI_FUNCTION	0x0
 
+#define VIDDID_CR		0
```

Comment:
It is impossible to find public PPR with description of this register, but I believe `CR` in `MEMPROT_CR` stands for `control register`, so it probably doesn't apply to `VIDDID` and should be dropped.

Reply:
Renamed: https://github.com/TrenchBoot/secure-kernel-loader/compare/951f504307833b994a00879e7142e33e43c8ee6c..18f535843b22e784dbbc8e1ec11d8dc099b3d0fe#diff-3d84071b8ad47879315e64cf64c07567330812a2cd339e0c845f8194c72aee54L40